test/integration/redis-cert/redis_cert.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+redis_cert_test() {
+    echo "Running redis-cert test"
+    update_with_file "redis-cert/install.yaml"
+    make_install
+    update_with_file "redis-cert/update.yaml"
+    make_upgrade
+}
```

Comment:
nit: EOF missing newline

Also, this might have the same considerations wrt chmod +x as above